src/openms/include/OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h

```diff
@@ -49,6 +49,13 @@ namespace OpenMS
       /// default constructor
       SiriusAdapterAlgorithm();
 
+      struct OPENMS_DLLAPI SiriusWorkspaceIndex
```

Comment:
technically, you can hide this implementation detail by just defining it in the cpp.
No need to do it here.